src/humancellatlas/data/metadata/api.py

```diff
@@ -582,10 +583,12 @@ class ImagingPreparationProtocol(Protocol):
     pass
 
 
-@dataclass
+@dataclass(init=False)
 class ManifestEntry:
+    json: MutableMapping = field(repr=False)  # should be MutableJSON
```

Comment:
If it should be, then why aren't you making it so? Maybe the comment should explain *that*.